rln-wasm/www/README.md

```diff
@@ -0,0 +1,20 @@
+# RLN WASM EXAMPLE APP
```

Comment:
Oh I see there's a README here already! I missed it because I expected it to be in the rln-wasm dir

Reply:
This example app will also be removed, and will exist in `https://github.com/waku-org/js-rln/`  :)